src/EventStore.Core/TransactionLog/Chunks/TFChunkManager.cs

```diff
@@ -289,6 +303,8 @@ public async ValueTask AddChunk(TFChunk.TFChunk chunk, CancellationToken token)
 			_chunksLocker.Release();
 		}
 
+		OnChunkSwitched?.Invoke(newChunk.ChunkInfo);
```

Comment:
do we want to call this even if the chunk switching failed?

Reply:
good catch! fixed